docs/source/conf.py

```diff
@@ -84,6 +84,10 @@
 # The short X.Y version.
 v_major, v_minor = LooseVersion(release).version[:2]
 version = "{}.{}".format(v_major, v_minor)
+# Cocotb commit ID
+commit_id = (
```

Comment:
I think it would be good to also (still) support building the documentation from a tarball without git installed.